docs/src/content/docs/fr/resources/plugins.mdx

```diff
@@ -141,7 +136,17 @@ Les [modules d'extension](/fr/reference/plugins/) peuvent personnaliser la confi
 	<LinkCard
 		href="https://github.com/delucis/starlight-llms-txt"
 		title="starlight-llms-txt"
-		description="Ajouter des fichiers llms.txt à votre site de documentation basé sur llmstxt.org"
+		description="Ajouter des fichiers llms.txt à votre site de documentation basé sur llmstxt.org."
+	/>
+	<LinkCard
+		href="https://github.com/trueberryless-org/starlight-toc-overview-customizer"
+		title="starlight-toc-overview-customizer"
+		description="Personnaliser la table des matières de Starlight avec un libellé de vue d'ensemble personnalisable."
```

Comment:
Re-using the built-in French translation for ["overview"](https://github.com/withastro/starlight/blob/main/packages/starlight/translations/fr.json#L15).